src/dcap_provider.h

```diff
@@ -68,6 +68,24 @@ typedef sgx_plat_error_t (*sgx_ql_get_revocation_info_t)(
 typedef void (*sgx_ql_free_revocation_info_t)(
     sgx_ql_revocation_info_t* p_revocation_info);
 
+/*****************************************************************************
+ * Data types and interfaces for getting qe identity info 
+ ****************************************************************************/
+
+typedef struct _sgx_qe_identity_info_t

```

Comment:
Please document that the buffers are null-terminated.

Reply:
good point.

Reply:
added

Reply:
Thanks for adding the docs above. Please be sure to also indicate `qe_id_info` and `issuer_chain` below are null terminated.

Reply:
added